public/ng-views/maps.html

```diff
@@ -512,23 +577,29 @@ <h5 ng-show="variants[tabnav].sel">
             {{ scenario[tabnav].offpeak.min | avgWaitTime }}
           </td>
           <td>
-            <span class="badge pull-right" ng-click="createVar = !createVar">
+            <!-- <span class="badge pull-right" ng-click="createVar = !createVar">
```

Comment:
This is all bad, and makes code difficult to read - please remove the vast swaths of commented out code - it's also dangerous as angular could misread them.